plugins/catalog-backend/src/database/DatabaseManager.ts

```diff
@@ -60,17 +61,40 @@ export class DatabaseManager {
   }
 
   public static async createTestDatabase(): Promise<Database> {
-    const knex = Knex({
+    const config: Knex.Config<any> = {
+      /*
+      client: 'pg',
```

Comment:
Did you want to keep this until after you've finished the debugging?

Reply:
Exactly, not pretty but temporarily useful / informative. Should probably be cleaned up later when things stabilise.